1-JavaScript/Week3/homework/ex3-tellFortune.js

```diff
@@ -30,28 +30,64 @@ body, this code is now written once only in a separated function.
 -----------------------------------------------------------------------------*/
 const numKids = [
   // TODO add elements here
+  2,
+  3,
+  1,
+  4,
+  5,
 ];
```

Comment:
It's also okay to have these arrays on one line:
```suggestion
const numKids = [2, 3, 1, 4, 5];
```